src/components/Dropdown/Dropdown.jsx

```diff
@@ -229,7 +229,7 @@ const Dropdown = ({
         Menu,
         ClearIndicator,
         Input,
-        ...(finalOptionRenderer && { Option }),
+        Option,
```

Comment:
is this by purpose? why?

Reply:
I just not find here where we pass finalOptionRenderer to the option component so I am not sure I am understand. can you show me?